espnet/bin/asr_enhance.py

```diff
@@ -12,9 +12,14 @@
 
 
 def main(args):
-    parser = argparse.ArgumentParser(
-        formatter_class=argparse.ArgumentDefaultsHelpFormatter)
+    parser = configargparse.ArgumentParser(
+        config_file_parser_class=configargparse.YAMLConfigFileParser,
+        formatter_class=configargparse.ArgumentDefaultsHelpFormatter)
     # general configuration
+    parser.add('--config', is_config_file=True, help='config file path')
+    parser.add('--config2', is_config_file=True, help='config file path')
```

Comment:
what these `config2` `config3` for?

Reply:
In some use case, someone would like to write only the difference from the base configuration in another configuration file.

```bash
asr-train.py --config conf/train.yaml --config2 conf/new.yaml
```
The options can be overwritten by the latter configuration file.

Reply:
gotcha. this is more expressive
```python
parser.add('--config2', is_config_file=True, help='second config file path that overwrites the settings in `--config`.')
parser.add('--config3', is_config_file=True, help='third config file path that overwrites the settings in `--config` and `--config2`.')
```

Reply:
I added as your comment.